js/interactive_background.js

```diff
@@ -0,0 +1,152 @@
+/* ===========================================================
+ * jquery-interactive_bg.js v1.0
+ * ===========================================================
+ * Copyright 2014 Pete Rojwongsuriya.
+ * http://www.thepetedesign.com
+ *
+ * Create an interactive moving background
+ * that reacts to viewer's cursor
+ *
+ * https://github.com/peachananr/interactive_bg
+ *
+ * License: GPL v3
+ *
+ * ========================================================== */
+
+!function($){
+
+  var defaults = {
+    strength: 25,
+    scale: 1.05,
+    animationSpeed: "100ms",
+    contain: true,
+    wrapContent: false
+  };
+
+  $.fn.interactive_bg = function(options){
+    return this.each(function(){
```

Comment:
This function has too many statements. (13)<br>This function's cyclomatic complexity is too high. (6)